python/ray/tune/tests/test_torch_trainable.py

```diff
@@ -171,6 +171,11 @@ def test_colocated_gpu_double(ray_4_node_gpu):  # noqa: F811
     trainable.stop()
 
 
+def test_validate_session():
```

Comment:
Make sure to use fixtures so that we don't leak state across tests.

Reply:
Done